src/Arrays/problems.md

```diff
@@ -1,11 +0,0 @@
-# Recommended LeetCode questions
-- Two Sum
-- Best Time to Buy and Sell Stock
-- Contains Duplicate
-- Product of Array Except Self
-- Maximum Subarray
-- Maximum Product Subarray
```

Comment:
Was this file created by you or was it there in the repo?

Reply:
This was already there, I guess

Reply:
> This was already there, I guess

Maybe add it back then.